langchain/src/util/zod.ts

```diff
@@ -0,0 +1,70 @@
+import { z, util } from "zod";
+
+export function printZodSchema(schema: z.ZodTypeAny, depth = 0): string {
```

Comment:
We've actually replaced this now in #1025

Reply:
Removed and am now using the `zodToJsonSchema` method